### ISSUE TITLE
The recognition accuracy is low

### ISSUE THREAD

**Issue by wxjames** [GuangmingZhu/AttentionConvLSTM#21]
Hi Dr. Zhu,

Thanks for your contributions. When I used your code to train the model, I found the training accuracy increased a little with the rise of epochs while the validation accuracy seemed not to change at different epochs. But both the training accuracy and the validation accuracy were no more than 0.2, which were too low. 

May I ask whether there are any methods to solve this problem?
![Screenshot from 2020-06-18 13-14-43](https://user-images.githubusercontent.com/33444066/85057276-2bedb280-b166-11ea-8dae-585dc859d03b.png)
